src/+replab/CommutantVar.m

```diff
@@ -282,8 +282,8 @@
                                       1  0];
 
                             % The code below is valid under the following condition
-                            assert(isequal(cat(3, basis1, basis2), replab.DivisionAlgebra('complex').basis));
-                            
+                            assert(isequal(cat(3, basis1, basis2), replab.DivisionAlgebra('C->R').basis));
+
```

Comment:
@jdbancal : I've merged that code, note that the DivisionAlgebra conventions have been renamed to express better what the encoding does. All tests run and the change is minimal.